### PR TITLE
Dartdoc output directory, pubspec.lock comment disambiguation

### PR DESCRIPTION
**Pull request by ArgentiApparatus**
1. Added directory for output of dartdoc tool.
2. Made style of comment for the pubspec.lock pattern consistent with others and disambiguated it.

### CODE REVIEW

File: Dart.gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Don’t commit the following directories created by pub.
+﻿# Don’t commit the following directories created by pub.
```
Comment (shiftkey) [inline Dart.gitignore]: :lipstick: drop this whitespace

**Comment by ArgentiApparatus** [comment]
I don't think I added whitespace to line 1, I'm really not sure why GitHub thinks that line is changed.

**Comment by ArgentiApparatus** [comment]
I believe issue with line 1 is now fixed.

**Comment by shiftkey** [comment]
Thanks!

**Pull request merged**